client/src/pages/LogInPage/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Button from '@/lib/design/Button';
+import { Styled } from './style';
+import { LogInForm } from '@/components/LogInForm';
+
+const LogInPage = () => {
+  const onClickSignIn = () => {
+    window.location.href = `/signup`;
```

Comment:
이 부분 새로고침돼서 리액트스럽지 못한 것 같습니다.
```suggestion
import { useHistory } from "react-router-dom";
const LogInPage = () => {
  const history = useHistory();
  const onClickSignIn = () => {
    history.push("/signup")
```
이렇게 리액트에서 제공하는 hook을 사용하는 것은 어떨까요?? 근데 SignIn버튼인데 로그인되는 게 아니라 회원가입 페이지로 가는 것 같은데 혹시 로그인 버튼은 어디있나요??